src/creator.html

```diff
@@ -46,7 +47,6 @@ <h1 id="title" ng-click="showTitleEditor = true">{{title}}</h1>
 				<p>Widget Scoring Mode</p>
 				<button type="button" class="header-options-button" ng-click="showScoreModeDialog = true; showBackgroundCover = true">{{scoreMode}}</button>
 			</div>
-			<!-- <button type="button" class="temp-qset-gen-button" ng-click="onSaveClicked('publish')">Simulate Publish</button> -->
```

Comment:
Generally I like removing unnecessary commented code, but I actually like keeping this one here. It's very useful for debugging.